crates/red_knot_test/src/lib.rs

```diff
@@ -72,21 +76,31 @@ pub fn run(
                         source_map.to_absolute_line_number(relative_line_number);
 
                     for failure in failures {
-                        let line_info =
-                            format!("{relative_fixture_path}:{absolute_line_number}").cyan();
-                        println!("  {line_info} {failure}");
+                        match output_format {
+                            OutputFormat::Cargo => {
```

Comment:
I'd prefer using `Concise` or `Full` or `Cli`. It isn't a format that's consumed by cargo. It's the CLI output

Reply:
Yeah, I also considered `Default`. `Cli` works for me!